### PR TITLE
Update example for ECS exec

### PR DESCRIPTION
**Pull request by jaezeu**

### CODE REVIEW

File: examples/fargate/data.tf
Comment (wayne-root) [inline examples/fargate/data.tf:1]: Do we need the following policies as well for exec? 
```
ssmmessages:CreateControlChannel
ssmmessages:CreateDataChannel
ssmmessages:OpenControlChannel
ssmmessages:OpenDataChannel
```

Comment (jaezeu) [reply examples/fargate/data.tf:1]: yup, but they are part of the arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore policy

File: examples/fargate/data.tf [hunk 1]
```diff
@@ -31,17 +31,24 @@ data "aws_iam_policy_document" "execution_custom_policy" {
...
-data "aws_iam_policy_document" "task_custom_policy" {
+data "aws_iam_policy_document" "task_ecs_exec_policy" {
   statement {
-    sid = "CustomTaskPolicy"
-
     actions = [
-      "s3:Get*",
-      "s3:List*",
+      "kms:Decrypt",
+      "kms:GenerateDataKey",
```
Comment (wayne-root) [inline examples/fargate/data.tf:38]: Can be removed

**Review by wayne-root — APPROVED** [review]
LGTM

**Pull request merged**